torch/_inductor/wrapper_benchmark.py

```diff
@@ -0,0 +1,169 @@
+import dataclasses
```

Comment:
Content of this file is split out from utils.py since that file is getting too large.